features/bevragen/persoon/partner/dev/fields-gba.feature

```diff
@@ -19,6 +19,7 @@ Functionaliteit: partner velden vragen met fields (persoon)
   Scenario: naam veld(en) van partner is/zijn in onderzoek
     Gegeven de persoon met burgerservicenummer '000000012' heeft een 'partner' met de volgende gegevens
     | naam                            | waarde                    |
+    | geslachtsnaam (02.40)           | .                         |
```

Comment:
Wat is de reden voor het toevoegen van dit gegeven? Als het wordt weggelaten, slagen de scenario's ook. Het kan ook zijn dat ik niet zie waar deze aanpassing past binnen de gegeven context in de p.r.

Reply:
wanneer er voor de partner geen enkel gegeven in groepen 1, 2, 3, 4, 6, 7, of 15 een waarde heeft, moet de partner niet worden geleverd. Dat het scenario nu wel slaagt zonder de geslachtsnaam is een bug.

Misschien had ik deze toevoeging eigenlijk in #1655 moeten doen...